sync/src/relayer/error.rs

```diff
@@ -42,4 +42,6 @@ pub enum Ignored {
     AlreadyInFlight,
     #[fail(display = "Already stored")]
     AlreadyStored,
+    #[fail(display = "Block is too early")]
+    TooEarlyBlock,
```

Comment:
I think `TooOldBlock` is more accurate. Too early can be interpreted that the block is in the future, and comes too early.

Reply:
fixed